IsraelHiking.Web/src/scss/common.scss

```diff
@@ -98,19 +98,32 @@ body {
         padding: 0 !important;
         display: block;
         width: v.$app-button-size;
-        height: v.$app-button-size;
+        height: 44px;
 
         & i {
             vertical-align: middle !important;
         }
     }
+    button.active:not(button.active:first-child , .search button.active ){
```

Comment:
Can a class be used here instead of this obscure css definition?

Reply:
I will update this with classes only.